flutter/pubspec.lock

```diff
@@ -21,10 +21,10 @@ packages:
     dependency: transitive
     description:
       name: async
-      sha256: "758e6d74e971c3e5aceb4110bfd6698efc7f501675bcfe0c775459a8140750eb"
+      sha256: d2872f9c19731c2e5f10444b14686eb7cc85c76274bd6c16e1816bff9a3bab63
       url: "https://pub.dev"
     source: hosted
-    version: "2.13.0"
+    version: "2.12.0"
```

Comment:
You are downgrading some package versions in the lock file! Is there a reason for that?
Please make sure you have the latest versions for dart and flutter before running `flutter pub get`.